lang/src/org/partiql/lang/eval/visitors/SelectListItemAliasVisitorTransform.kt

```diff
@@ -0,0 +1,63 @@
+package org.partiql.lang.eval.visitors
+
+import org.partiql.lang.domains.extractSourceLocation
+
+import org.partiql.lang.domains.PartiqlAst
+import org.partiql.lang.eval.extractColumnAlias
+import org.partiql.pig.runtime.SymbolPrimitive
+
+
+/**
```

Comment:
This class performs exactly the same function as `SelectListItemAliasRewriter` but works on the PIG-generated `PartiqlAst` instead of `ExprNode`.